src/xref2/link.ml

```diff
@@ -262,8 +262,19 @@ and comment_tag env parent ~loc:_ (x : Comment.tag) =
       `Deprecated (comment_nestable_block_element_list env parent content)
   | `Param (name, content) ->
       `Param (name, comment_nestable_block_element_list env parent content)
-  | `Raise (name, content) ->
-      `Raise (name, comment_nestable_block_element_list env parent content)
+  | `Raise ((`Reference (r, reference_content) as orig), content) -> (
+      assert (reference_content = []);
```

Comment:
We do not need to care. Either way is fine: use a more restricted type or accept every possible values.

Reply:
I've changed it to allow non-empty reference content, although that's still never created for raise tags. A more restricted type would've been slightly annoying since that wouldn't have allowed the reuse of the normal reference variant and all the code there already is to handle those.